app/views/users/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Create a new user</h1>

```

Comment:
- There should be two commented out forms before this one. Please add the two commented out forms below.
#### `form`
```ruby
<form action="users_path" method="post" accept-charset="UTF-8">
  <label for="user[username]">Username</label>
  <input name="user[username]" type="text" />
  <input
    type="hidden"
    name="authenticity_token"
    value="<%= form_authenticity_token %>"
  />
  <input type="submit" value="Submit" />
</form>
```

#### `form_tag`
```ruby
<%= form_tag(users_path) do %> <%= label_tag 'username' %> <%= text_field_tag
'username' %> <%= label_tag 'email' %> <%= email_field_tag 'email' %> <%=
label_tag 'password' %> <%= password_field_tag 'password' %> <%= submit_tag %>
<% end %>
```
### Note
- The aim of this PR is to make sure that you know who to create a form using one of the three ways